index.html

```diff
@@ -1,149 +1,166 @@
+        </li>
+        <li class="major-element level-3xxx" id="core-courses">Core Courses
+          <ul>
+            <li><span class="discipline csci">CSci</span> <span class="course-number">3402</span></li>
+            <li><span class="discipline csci">CSci</span> <span class="course-number">3403</span></li>
+            <li><span class="discipline csci">CSci</span> <span class="course-number">3501</span></li>
+            <li><span class="discipline csci">CSci</span> <span class="course-number">3601</span></li>
+          </ul>
+        </li>
+        <li class="major-element level-4xxx" id="electives">Ten credits of electives 4xxx so that the split of credits
+          is
+          at least 4 in two of the areas and at least 2 in the third area.
+        </li>
+        <li class="major-element level-1xxx" id="ethical-and-social-implications">
+          <span class="discipline is">IS</span> <span class="course-number">1091</span>
+        </li>
+        <li class="major-element level-4xxx" id="senior-seminar">
+          <span class="discipline csci">CSci</span> <span class="course-number">4901</span>
+        </li>
+        <li class="major-element" id="math-and-stats">12 credits of:
+          <ul>
+            <li><span class="discipline math">Math</span> at or above <span class="course-number">1101</span> excluding
+              "History of math"
+            </li>
+            <li><span class="discipline stat">Stat</span> 2xxx or above</li>
+          </ul>
+        </li>
+      </ul>
+    </div>
+  </div>
+  <div class="section" id="faculty">
+    <h2>Faculty</h2>
+    <ul id="faculty-list">
+      <li id="Elena">
+        <div class="name">Elena</div>
+        <div class="office">Science 2325</div>
+        <img class="pic" src="./assets/images/Elena.jpg" alt="Elena" />
+      </li>
+      <li id="KK">
+        <div class="name">KK</div>
+        <div class="office">Science 2525</div>
+        <img class="pic" src="./assets/images/KK.jpg" alt="KK" />
+      </li>
+      <li id="Nic">
+        <div class="name">Nic</div>
+        <div class="office">Science 1315</div>
+        <img class="pic" src="./assets/images/Nic.jpg" alt="Nic" />
+      </li>
+      <li id="Peter">
+        <div class="name">Peter</div>
+        <div class="office">Science 1440</div>
+        <img class="pic" src="./assets/images/Peter.jpg" alt="Peter" />
+      </li>
+    </ul>
+  </div>
 
-     </ul>
-   </div>
+  <div class="card" id="validation">
+    <div class="card-body">
+      <h2 class="card-title">Validation</h2>
+      <p class="card-text">
+        In general you wouldn't want to put links like these
+        (or <a
+          href="https://stackoverflow.com/questions/36383923/how-to-get-the-badge-when-validating-html-in-the-w3c-validator">
+          validation badges</a>) on real project web pages. We've included them because it makes it easier for us to
```

Comment:
We can probably make this <a> tag look a bit nicer using the short form of the StackOverflow url: 🙂

```suggestion
        In general you wouldn't want to put links like these
        (or <a href="https://stackoverflow.com/q/36383923">validation badges</a>)
        on real project web pages. We've included them because it makes it easier for us to
```

Reply:
Or alternatively, if you want to keep the long form: 🙂
```suggestion
        In general you wouldn't want to put links like these (or
        <a href="https://stackoverflow.com/questions/36383923/how-to-get-the-badge-when-validating-html-in-the-w3c-validator">
          validation badges
        </a>) on real project web pages. We've included them because it makes it easier for us to
```